src/main/java/uk/gov/hmcts/ccd/domain/service/getevents/DefaultGetEventsOperation.java

```diff
@@ -30,7 +31,7 @@
 
     @Autowired
     public DefaultGetEventsOperation(CaseAuditEventRepository auditEventRepository, @Qualifier(
-        CreatorGetCaseOperation.QUALIFIER) final GetCaseOperation getCaseOperation, UIDService uidService) {
+        AccessControlledGetCaseOperation.QUALIFIER) final GetCaseOperation getCaseOperation, UIDService uidService) {
```

Comment:
this seems wrong. It's the other way round, `AccessControlledGetCaseOperation` should eventually wrap and call `DefaultGetEventsOperation`

Reply:
DefaultGetEventsOperation had a CreatorGetCaseOperation composed only to decide if we can access the case and if it exists. We are in 'GetEvents' so not sure how this could be other way around. Those don't share the same interface.